js/provider/taihe.js

```diff
@@ -283,13 +284,13 @@ function build_taihe() {
     };
   }
 
-  function th_show_playlist(url) {
+  static show_playlist(url) {
     const offset = Number(getParameterByName('offset', url));
     const subCate = getParameterByName('filter_id', url);
-
+    console.log(this.axiosTH);
```

Comment:
extra console?

Reply:
https://github.com/mikelxk/listen1_chrome_extension/blob/6feb5f32757abf7fc685aadd579211d9c230ced1/js/provider/taihe.js#L30-L32
I used a static class property which isn't quite supported by eslint and I think it's really a burden to add a babel parser.